src/ui/common/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@aqueducthq/common",
-  "version": "0.0.8",
+  "version": "0.0.9",
```

Comment:
Latest version here should be 0.0.10, I have another related PR that will probably go in before this one, so you will be able to just rebase and get latest changes.

https://github.com/aqueducthq/aqueduct/pull/251